sky/utils/common_utils.py

```diff
@@ -228,6 +232,13 @@ def get_global_job_id(job_timestamp: str,
     return global_job_id
 
 
+def get_unique_task_name(_: 'task_lib.Task') -> str:
+    timestamp = int(time.time())
+    unique_suffix = uuid.uuid4().hex[:6]
+    name = f'task_{timestamp}_{unique_suffix}'
+    return name
```

Comment:
Lets respect the original task name and append uuid suffix to it.

Reply:
Well, this function is only called when task name is `None`. However, to keep each task name unique, I would do what you said and add the `uuid` suffix to every task name while keeping this logic when the task name is null.